test/test_c10d.py

```diff
@@ -1309,31 +1309,37 @@ def test_allgather_stress_cuda(self):
     def test_allgather_coalesced_checks(self):
         store = c10d.FileStore(self.file_name, self.world_size)
         pg = c10d.ProcessGroupGloo(store, self.rank, self.world_size, self.opts())
-        dummy_input = [torch.Tensor([1])]
+        dummy_input = [torch.zeros([1], dtype=torch.float32)]
```

Comment:
sorry -- why are these not translated exactly?

`torch.Tensor([1])` is `torch.ones([1])`, not zeros, right?

also same with the line below, why did that change from -1 to 0?

Reply:
This only tests error handling, so they underlying values here should not be important (all_gather_coalesced never copies anything in this function). I am happy to change it back if you prefer